url-gen/src/test/kotlin/com/cloudinary/AuthTokenTest.kt

```diff
@@ -165,7 +164,7 @@ class AuthTokenTest {
         val url =
             cloudinary.image().generate("sample.jpg")
         assertEquals(
-            "https://res.cloudinary.com/test123/image/upload/sample.jpg?__cld_token__=st=11111111~exp=11111411~hmac=ec2d0e9a87f6dd5147c36f39c5e88c28fdaa53d1e04f3d70160f7ebe62a60c7a&_a=$expectedAnalytics",
+            "https://res.cloudinary.com/test123/image/upload/sample.jpg?__cld_token__=st=11111111~exp=11111411~hmac=ec2d0e9a87f6dd5147c36f39c5e88c28fdaa53d1e04f3d70160f7ebe62a60c7a",
```

Comment:
test name `testAuthTokenWithAnalytics` contradicts assertion being made

I would keep the test and add another one to cover the new use-case.

Reply:
Not sure what you mean
We cant keep the current test as it'll fail , when auth token exist analytics will not appear on the URL
So I changed the current test to work with `analytics = true` but the assert is to make sure that the analytics string does not appear

Reply:
I think the idea is that the test tests the opposite

It's not "AuthTokenWithAnalytics" but rather "DisableAnalyticsWithAuthToken"